core/src/main/java/org/jruby/RubyTime.java

```diff
@@ -899,7 +899,7 @@ public RubyString asctime() {
     @JRubyMethod
     public IRubyObject to_s() {
         DateTimeFormatter simpleDateFormat = isUTC() ? TO_S_UTC_FORMATTER : TO_S_FORMATTER;
-
+        
```

Comment:
Remove rogue whitespace; formatting standards for our codebase have no whitespace on blank lines (other than the newline).